data/json/items/armor.json

```diff
@@ -5809,6 +5809,78 @@
     "material_thickness": 1,
     "flags": [ "WAIST", "OVERSIZE", "WATER_FRIENDLY" ]
   },
+  {
+    "id": "ammo_pouch",
+    "type": "ARMOR",
+    "name": "ammo pouch",
```

Comment:
Plural name needed.

Reply:
I do not agree it is a single item that its always separated from similar items. If you go into the code only armor that comes in pairs like "arm guards" have that line of text so I will not add it to this item for consistency sake.

Reply:
You are wrong. Game automatically adds a trailing `s` to the item's `name` if no `name_plural` is defined. So if we don't specifically set a plural name for arm guards, we'll get `arm guardss` in the game for several items. In your case not having a defined plural name will result in `ammo pouchs` for several items. Look in the `data\json\items\armor\ammo_pouch.json`. There are a lots of pouches with a defined plural name.

Reply:
So to clarify I need to set its plural to still be "ammo pouch"?

Reply:
ok I see my mistake I forgot pouch needs an "es" at the end, that is my bad sorry for doubting you